projects/online/prior.yaml

```diff
@@ -0,0 +1,83 @@
+class_path: amplfi.train.data.utils.utils.ParameterSampler
+init_args:
+  parameters:
+    chirp_mass: 
+      class_path: torch.distributions.Uniform
+      init_args:
+        low: 0 
+        high: 200 
+        validate_args: false
+    mass_ratio: 
+      class_path: torch.distributions.Uniform
+      init_args:
+        low: 0.0 
+        high: 0.999
+        validate_args: false
+    distance: 
+      class_path: torch.distributions.Uniform
+      init_args:
+        low: 0 
+        high: 3100 
```

Comment:
This should be higher, right?

Reply:
Yeah I did notice that - I made it 10000 here, we should confirm the same in the deployment configs

Reply:
Probably explains (some of) why that plot of injected vs recovered distance looks the way it does